docs/tails_printing_guide.rst

```diff
@@ -75,7 +75,7 @@ following:
 
 This indicates the Tails is attempting to install the USB printer; assuming you
 receive no errors in this process, you will then see the following screen,
-which indicates that the printer is "ready" for printing.
+which indicates that the printer is **ready** for printing.
```

Comment:
Not sure if this should be in bold.

Reply:
Agree. I think this emphasis should be removed.

Reply:
ok, will revert it back later today. thanks for the review.